docs/data/sdks/unity/index.md

```diff
@@ -4,13 +4,18 @@ description: The Amplitude Unity SDK installation and quick start guide.
 icon: material/unity
 ---
 
+
+![Unity SDK](https://img.shields.io/github/v/tag/amplitude/unity-plugin?label=Unity%20SDK)
+
 The Amplitude Analytics Unity SDK is a plugin to simplify the integration of Amplitude iOS and Android SDKs into your Unity project. This SDK works with Unity 2019.3.11 and higher.
 
 !!!info "SDK Resources"
     - [Unity SDK Repository :material-github:](https://github.com/amplitude/unity-plugin)
     - [Unity SDK Releases :material-code-tags-check:](https://github.com/amplitude/unity-plugin/releases)
 
-!!!note
+--8<-- "includes/no-ampli.md"
```

Comment:
I think we should also add the "no ampli" warning to Unreal as well.

Reply:
Added the no ampli warning. Do we still consider the unreal sdk as beta now?